### PR TITLE
reorganize BN-related "how it works" pages

### PR DESCRIPTION
**Pull request by r-birkner**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [ ] Follows the [developer docs style guide](style-guide.md).
- [ ] Follows the [best practices and guidelines](README.md#best-practices).
- [ ] New documentation pages include [document tags](README.md#document-tags).
- [ ] New documentation pages include [SEO keywords](README#seo-keywords).
- [ ] New documents are in the `.mdx` file format to support the previous two components. 
- [ ] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

**Comment by r-birkner** [comment]
@nathanosdev you might want to take a look at the webaccess/webserving part. This MR basically goes over this page https://internetcomputer.org/how-it-works/smart-contracts-serve-the-web/

**Review by yvonneanne — APPROVED** [review]
LGTM

File: how-it-works/6_webaccess/01_web_access.subpage.md [hunk 1]
```diff
@@ -7,55 +7,41 @@ slug: smart-contracts-serve-the-web
 
 # Smart Contracts serve the web
```
Comment (yvonneanne) [inline how-it-works/6_webaccess/01_web_access.subpage.md:8]:
```suggestion
```

File: how-it-works/6_webaccess/01_web_access.subpage.md [hunk 2]
```diff
@@ -7,55 +7,41 @@ slug: smart-contracts-serve-the-web
...
+The Internet Computer is the only blockchain that can host an entire dapp – frontend, backend and data.  This is a crucial and distinguishing feature allowing dapps to run 100% on-chain inheriting the security and decentralization of blockchain without sacrificing speed or affordability. Browsing dapps hosted on the Internet Computer is as seamless as browsing Web2 apps. In fact, this very site is hosted 100% on-chain, 100% on the Internet Computer and there are many more [dapps hosted on the Internet Computer that can be found here](https://internetcomputer.org/ecosystem/).
```
Comment (jessiemongeon1) [inline how-it-works/6_webaccess/01_web_access.subpage.md:10]:
```suggestion
The Internet Computer is the only blockchain that can host an entire dapp – frontend, backend and data.  This is a crucial and distinguishing feature allowing dapps to run 100% on-chain inheriting the security and decentralization of blockchain without sacrificing speed or affordability. Browsing dapps hosted on the Internet Computer is as seamless as browsing Web2 apps. In fact, this very site is hosted 100% on-chain on the Internet Computer and there are many more [dapps hosted on the Internet Computer that can be found here](https://internetcomputer.org/ecosystem/).
```

File: how-it-works/6_webaccess/03_boundary_nodes.subpage.md [hunk 3]
```diff
@@ -22,9 +20,9 @@ The HTTP gateway allows users to access the dapps hosted on the IC through their
...
+Both the HTTP gateway and the API boundary node are currently combined into the boundary node. Work to separate the two into two independent services is ongoing. The first proposal to deploy API boundary nodes under the control of the NNS has been adopted. The next step is to create the HTTP gateways, that can be run by anyone, and shift the traffic from the existing boundary nodes to the HTTP gateways and API boundary nodes. [For more information on the current state check the updates in the forum.](https://forum.dfinity.org/t/boundary-node-roadmap/15562)
 
-In addition to the two endpoints, the boundary nodes provide to access the IC, the boundary nodes also provide caching to improve the performance of the dapps hosted on the IC.
+In addition to the two endpoints, the boundary nodes provide to access the IC, the boundary nodes also provide caching to improve the performance and a [custom domains service](/docs/).
```
Comment (jessiemongeon1) [inline how-it-works/6_webaccess/03_boundary_nodes.subpage.md:25]:
```suggestion
In addition to the two endpoints, the boundary nodes provide to access the IC, the boundary nodes also provide caching to improve the performance and a [custom domains service](/docs/current/developer-docs/web-apps/custom-domains/using-custom-domains).
```

File: how-it-works/6_webaccess/01_web_access.subpage.md [hunk 4]
```diff
@@ -7,55 +7,41 @@ slug: smart-contracts-serve-the-web
...
 
-[Hosting a static website on the IC](/docs/current/samples/host-a-website/)
+The API boundary node simply takes the API canister call and forwards it to a replica node, which is part of the subnet that hosts the target canister.
 
-[HTTP Gateway Protocol](/docs/current/references/ic-interface-spec/#http-gateway)
+The canister receives that query call for its `http_request`-method, processes it and replies. To this end, the canister needs to implement the [Canister HTTP Interface](https://internetcomputer.org/docs/current/references/http-gateway-protocol-spec#canister-http-interface), which is part of the HTTP gateway protocol.
```
Comment (jessiemongeon1) [inline how-it-works/6_webaccess/01_web_access.subpage.md:38]:
```suggestion
The canister receives that query call for its `http_request`-method, processes it and replies. To this end, the canister needs to implement the [Canister HTTP Interface](/docs/current/references/http-gateway-protocol-spec#canister-http-interface), which is part of the HTTP gateway protocol.
```

File: how-it-works/6_webaccess/01_web_access.subpage.md [hunk 5]
```diff
@@ -7,55 +7,41 @@ slug: smart-contracts-serve-the-web
...
 
-[Serving web content](/capabilities/serve-web-content/)
+It all starts in the browser. The browser does not know that this site is hosted on the Internet Computer and makes a normal HTTP request, just as it would for any other site. It sends that request to the server hosting internetcomputer.org.
 
-[Building a front-end dapp on the IC](https://medium.com/dfinity/building-a-front-end-dapp-on-the-internet-computer-55985f0a595b)
+This server takes the HTTP request and translates it into an API canister call. In particular, it turns the HTTP request into a query call to the `http_request`-method of the target canister and puts the requested path, the HTTP request headers and the body into the payload of that query call. How this works in detail is specified in the [HTTP gateway protocol](https://internetcomputer.org/docs/current/references/http-gateway-protocol-spec). Today, there exist two main implementations of the [HTTP gateway protocol](https://internetcomputer.org/docs/current/references/http-gateway-protocol-spec): `icx-proxy`, a remote HTTP gateway, which runs on the boundary nodes; and the `IC HTTP proxy`, a local HTTP gateway, which can be downloaded and run locally.
```
Comment (jessiemongeon1) [inline how-it-works/6_webaccess/01_web_access.subpage.md:34]:
```suggestion
This server takes the HTTP request and translates it into an API canister call. In particular, it turns the HTTP request into a query call to the `http_request`-method of the target canister and puts the requested path, the HTTP request headers and the body into the payload of that query call. How this works in detail is explained in the [HTTP gateway protocol specification](/docs/current/references/http-gateway-protocol-spec). Today, there exist two main implementations of the [HTTP gateway protocol](/docs/current/references/http-gateway-protocol-spec): `icx-proxy`, a remote HTTP gateway, which runs on the boundary nodes; and the `IC HTTP proxy`, a local HTTP gateway, which can be downloaded and run locally.
```

**Review by jessiemongeon1 — APPROVED** [review]

File: how-it-works/6_webaccess/01_web_access.subpage.md (hunk 1 above)
Comment (r-birkner) [reply how-it-works/6_webaccess/01_web_access.subpage.md:8]: I don't understand this change: Should we remove the title of the page?

Comment (yvonneanne) [reply how-it-works/6_webaccess/01_web_access.subpage.md:8]: The title is already above, there is no point in having it appear twice
image

**Pull request closed without merging**